engine/gameobject/src/gameobject/gameobject.cpp

```diff
@@ -493,7 +503,7 @@ namespace dmGameObject
     {
         if (max_instances > INVALID_INSTANCE_INDEX)
         {
-            dmLogError("max_instances must be less or equal to %d", INVALID_INSTANCE_INDEX);
+            dmLogError("max_instances must be less or equal to %d", INVALID_INSTANCE_INDEX - 1);
```

Comment:
It was a mistake in log. `max_instances` can't be equal `INVALID_INSTANCE_INDEX`